web/src/engine/websites/CoroCoro.ts

```diff
@@ -25,4 +63,69 @@ export default class extends DecoratableMangaScraper {
     public override get Icon() {
         return icon;
     }
+
+    public override async FetchMangas(provider: MangaPlugin): Promise<Manga[]> {
+        const scripts = await FetchCSS<HTMLScriptElement>(new Request(new URL('/rensai', this.URI)), 'script');
+        const json = this.FindAndParseJSON(scripts, 'totalChapterLikes');
+        const { weekdays: { mon, tue, wed, thu, fri, sat, sun } } = this.LocateObjectWithNamedProperty<JSONMangas>(json, 'weekdays');
+        return [mon, tue, wed, thu, fri, sat, sun].reduce((accumulator: Manga[], day) => {
+            const mangas = day.map(manga => new Manga(this, provider, `/title/${manga.id}`, manga.name));
+            accumulator.push(...mangas);
+            return accumulator;
+        }, []);
+
+    }
+
+    public override async FetchChapters(manga: Manga): Promise<Chapter[]> {
+        const scripts = await FetchCSS<HTMLScriptElement>(new Request(new URL(`${manga.Identifier}`, this.URI)), 'script');
+        const json = this.FindAndParseJSON(scripts, 'omittedMiddleChapters');
```

Comment:
The `json` variable may be undefined here, is this intentional?

Reply:
variable **value** may be undefined, or null, or whatever but the variable itself is always present afaik.